src/Behat/Testwork/Output/Printer/Factory/OutputFactory.php

```diff
@@ -31,10 +31,8 @@ abstract class OutputFactory
      * @var array
      */
     private $outputStyles = [];
-    /**
-     * @var bool|null
-     */
-    private $outputDecorated = null;
+
+    private ?bool $outputDecorated = null;
```

Comment:
The `setOutputDecorated` function does not have a strict type on the input parameter. Is there any chance this could cause a BC break if people are calling it with a non-boolean value?